frameworks/PHP/symfony/config/packages/doctrine.yaml

```diff
@@ -1,22 +1,17 @@
-parameters:
-    # Adds a fallback DATABASE_URL if the env var is not set.
-    # This allows you to run cache:warmup even if your
-    # environment variables are not available yet.
-    # You should not need to change this value.
-    env(DATABASE_URL): ''
-
 doctrine:
```

Comment:
The production ready config for doctrine seems to be missing so that the metadata is cached: https://github.com/symfony/recipes/blob/master/doctrine/doctrine-bundle/1.6/config/packages/prod/doctrine.yaml

Reply:
It is allowed the metadata cache.

Reply:
on this container it's request time about the same and sometimes even slower what this production ready config is enabled?!

Reply:
That would be strange and a sign that something is not correctly configured.